hwdb/20-usb-vendor-model.hwdb

```diff
@@ -2142,7 +2142,7 @@ usb:v03FE*
  ID_VENDOR_FROM_DATABASE=Farallon Comunications
 
 usb:v0400*
- ID_VENDOR_FROM_DATABASE=National Semiconductor Corp.
+ ID_VENDOR_FROM_DATABASE=National
```

Comment:
"Semiconductor" should not be dropped, "Corp." can be dropped but doesn't have to.